### PR TITLE
feature: Fix RPs layout for mobile devices

### PR DESCRIPTION
**Pull request by annacv**
# Pull request template

Describe the purpose of the change, the specific changes done in detail, and the issue you have fixed.

## Motivation and context

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [ ] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](https://github.com/empathyco/x/blob/main/.github/contributing/tests.md):

## Checklist:

- [ ] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

File: src/components/main.vue [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <LocationProvider location="results">
       <Results />
     </LocationProvider>
-    <LocationProvider v-if="showRelatedPrompts" location="related_prompts">
+    <LocationProvider v-if="showRelatedPrompts" :location="semanticsLocation">
```
Comment (victorcg88) [inline src/components/main.vue:7]: Is this intentional? Is it correct?

File: package-lock.json [hunk 2]
```diff
@@ -1229,7 +1229,7 @@
     },
     "node_modules/@clack/prompts/node_modules/is-unicode-supported": {
       "version": "1.3.0",
-      "extraneous": true,
+      "dev": true,
```
Comment (victorcg88) [inline package-lock.json:1232]: Why is this being reverted?

**Review by victorcg88 — COMMENTED** [review]
IMHO, the top and the bottom margins of the prompts could look better if they are the same. WDYT?

image

Also, I'm wondering if we should add still a bit more space to the bottom, so the prompt is clearly visible

image

But this is subjective, maybe is worth to ask the rest of the team.

File: package-lock.json (hunk 2 above)
Comment (annacv) [reply package-lock.json:1232]: Hi, I thought I had uploaded this file too, sorry

File: src/components/main.vue (hunk 1 above)
Comment (annacv) [reply src/components/main.vue:7]: Yep, related_prompts should be added as a location value in x-components if we want to use it here,
maybe it should also be discussed with the team @victorcg88

**Review by victorcg88 — APPROVED** [review]

**Pull request merged**